scripts/merge-measures-data.js

```diff
@@ -114,5 +114,29 @@ function mergeQpp(qppJson) {
 
   console.error('did not find measure details for the following', measuresNotFound);
 
+  enrichACIMeasures(qppJson);
+
   return JSON.stringify(qppJson, null, 2);
 }
+
+
+/**
+ * Will add extra metadata to ACI measures
+ * @param measures - the measures to enrich
+ */
+function enrichACIMeasures(measures) {
+
+  // load the relations from file
+  var aciRelations = require('../util/aci-measure-relations.json');
+
+  measures
+      .filter(m => m.category === 'aci')
```

Comment:
Sure!, it is a good idea to create an [editor config file](http://editorconfig.org) so that formatting will be easier for internal devs and external contributors. I am adding one, configured it to have 2 spaces, but please adjust it to match other preferences later.